src/Illuminate/Database/Console/Migrations/MigrateCommand.php

```diff
@@ -49,7 +49,7 @@ public function __construct(Migrator $migrator)
 	 */
 	public function fire()
 	{
-		if ( ! $this->confirmToProceed()) return;
+		if ( ! $this->confirmToProceed("About to run ".$this->name)) return;
```

Comment:
By changing line 16 in ConfirmableTrait.php to this:

    $warning .= 'About to run '.$this->name.'. Are you sure you want to do this in production mode?';